README.md

```diff
@@ -0,0 +1,127 @@
+# Mayflower-opinionated Nix modules, overlay and Hydra Configuration
+
+This is a collection of too opinionated or too experimental Nix code we use at
+Mayflower, mostly in production.
+
+## Overlay
+
+The overlay consists of some additional packages that are too experimental to
+upstream currently:
+ * mailman3, including Postorius and Hyperkitty
+ * serviceOverview, see below in the modules section
+ * dovecot and postfix, overridden to include PgSQL support
+
+**Using the overlay**
+
+When importing `nixpkgs`:
+```
+let
```

Comment:
To make it easier for a new user to get started, would you mind adding some nixpkgs import line such as `{ pkgs ? import <nixpkgs> { } }:` or am I missing something?